openrtx/include/protocols/M17/M17Constants.hpp

```diff
@@ -37,12 +37,15 @@ static constexpr size_t M17_FRAME_SYMBOLS    = 192;
 static constexpr size_t M17_SYNCWORD_SYMBOLS = 8;
 static constexpr size_t M17_FRAME_BYTES      = M17_FRAME_SYMBOLS / 4;
 
-static constexpr syncw_t LSF_SYNC_WORD    = {0x55, 0xF7};  // LSF sync word
-static constexpr syncw_t BERT_SYNC_WORD   = {0xDF, 0x55};  // BERT data sync word
-static constexpr syncw_t STREAM_SYNC_WORD = {0xFF, 0x5D};  // Stream data sync word
-static constexpr syncw_t PACKET_SYNC_WORD = {0x75, 0xFF};  // Packet data sync word
-static constexpr syncw_t EOT_SYNC_WORD    = {0x55, 0x5D};  // End of transmission sync word
+static constexpr syncw_t LSF_SYNC_WORD    = {0x55, 0xF7}; // LSF sync word
+static constexpr syncw_t BERT_SYNC_WORD   = {0xDF, 0x55}; // BERT data sync word
+static constexpr syncw_t STREAM_SYNC_WORD = {0xFF,
```

Comment:
Why did the lint break this? Again for the 80 column overflow?

Reply:
Yes, but we can force the comment to wrap instead.